docs/sdk-guide.md

```diff
@@ -478,6 +489,16 @@ public CarbonWriter build() throws IOException, InvalidLoadOptionException;
 ```
 
 ### Class org.apache.carbondata.sdk.file.CarbonWriter
+
```

Comment:
why we changed the order?

Reply:
because it should builder, build, and then colse when user use the API, so it's better that the doc should keep the same order